ellipsoid ellipsoid
float64 steps
---
point[] points
